test/PackageUtilities.js

```diff
@@ -77,10 +77,18 @@ describe("PackageUtilities", () => {
       });
     });
 
-    it("should throw when --scope is given but empty", () => {
-      assert.throws(() => {
-        PackageUtilities.filterPackages(packages, "");
-      });
+    it("should return everything when --scope is given but empty", () => {
+      assert.deepEqual(
+        PackageUtilities.filterPackages(packages, "").map((pkg) => pkg.name),
+        ["package-3", "package-4", "package-a-1", "package-a-2"]
+      );
+    });
```

Comment:
This is an change in edge-case behavior due to the way filtration is factored to support hoisting as well as scope/ignore.  Previously `--scope=` was an error.  Now it's a noop.